torchci/README.md

```diff
@@ -41,3 +41,18 @@ them to your local server.
 We use [Vercel](https://vercel.com/torchci) as our deployment platform. Pushes
 to `main` and any other branches will automatically be deployed to Vercel; check out
 the bot comments for how to view.
+
+## How to edit a Rockset query lambdas
+
+The source of truth for our query lambdas is in `rockset/`. We use the Rockset
```

Comment:
I see you included an empty rockset/README.md. 

Is that intended?

Reply:
oops